packages/fast-element/src/ast.ts

```diff
@@ -0,0 +1,41 @@
+export interface IExpressionSource {

```

Comment:
I would love to pull the "I" from the interfaces here - it's a convention we've held to with FAST and one I'd prefer to keep if possible. Thoughts on that? I'm happy to create an issue to track that work.

Reply:
No objections from me. Do you want to do it as part of this PR or as a follow-up?

Reply:
I think we can do it as a follow-up. I'll write up an issue.